### PR TITLE
Add DashboardTabFlakinessAlertOptions to config.proto

### PR DESCRIPTION
**Pull request by RonWeber**
Add a configuration setting for the flakiness alerter (Testgrid will eventually alert about flaky tests).

### CODE REVIEW

**Review by michelle192837 — APPROVED** [review]

File: pb/config/config.proto [hunk 1]
```diff
@@ -543,6 +543,9 @@ message DashboardTab {
   // Configuration options for dashboard tab alerts.
   DashboardTabAlertOptions alert_options = 15;
 
+  // Configuration options for dashboard tab flakiness alerts.
+  DashboardTabFlakinessAlertOptions flakiness_alert_options = 24;
```
Comment (e-blackwelder) [inline pb/config/config.proto:547]: Is there a reason to not put this in numerical order? (I would expect future maintainers to simply scroll down to the end of the message and say "looks like 23 is the last one".)

Comment (RonWeber) [reply pb/config/config.proto:547]: It seemed unintuitive to separate alert_options and flakiness_alert_options.

**Review by michelle192837 — APPROVED** [review]

**Pull request merged**